Xamarin.Forms.Platform.iOS/Controls/Snackbar/SnackBar.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using Xamarin.Forms.Platform.iOS.Controls.Snackbar.SnackbarViews;
+using Foundation;
+using System.Threading.Tasks;
+using UIKit;
+
+namespace Xamarin.Forms.Platform.iOS.Controls.Snackbar
+{
+	public class SnackBar
```

Comment:
Does it make sense to call it SnackBar? or should we call it Toast?

Reply:
I am going to rename the method to DisplaySnackbar. Snack bar = Toast + action button. I hide action button if action is null or action button text is empty